lib/manageiq/loggers/cloud_watch.rb

```diff
@@ -7,11 +7,11 @@ module Loggers
     class CloudWatch < Base
       NAMESPACE_FILE = "/var/run/secrets/kubernetes.io/serviceaccount/namespace".freeze
 
-      def self.new(*args)
-        access_key_id     = ENV["CW_AWS_ACCESS_KEY_ID"].presence
-        secret_access_key = ENV["CW_AWS_SECRET_ACCESS_KEY"].presence
-        log_group_name    = ENV["CLOUD_WATCH_LOG_GROUP"].presence
-        log_stream_name   = ENV["HOSTNAME"].presence
+      def self.new(*args, access_key_id: nil, secret_access_key: nil, log_group: nil, hostname: nil)
```

Comment:
Gah - I merged it and just noticed a bug

- the params here are log_group and hostname, but the variables being `||=` below are log_group_name and log_stream_name

Reply:
Can you make a PR to fix and also possibly add a spec?

Reply:
I'm already out, is it enough if I do it tomorrow?

Reply:
Yeah tomorrow is fine